hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeIntegrationTest.java

```diff
@@ -163,5 +167,75 @@ public void testCacheMetrics() {
                 .withQueryParam("cawact", equalTo("1")));
     }
 
+    @Test
+    public void testMapLatencies() {
+        DelayMapStore mapStore = new DelayMapStore();
+        MapStoreConfig mapStoreConfig = new MapStoreConfig();
+        mapStoreConfig.setEnabled(true);
+        mapStoreConfig.setClassName(getClass().getName());
```

Comment:
What does this do? It doesn't seem to be necessary (actually it doesn't cause any exceptions only because the following setImplementation() call will set the className field to null)